index.html

```diff
@@ -701,8 +708,107 @@
 Many of the [=principles=] described in the rest of this document extend the [=user agent=]'s duties and
 make them more precise.
 
+## Balancing Different Privacy Principles {#balancing}
+
+While privacy principles are designed to work together and support each other,
+occasionally a proposal to improve how a system follows one privacy principle will reduce
+how well it follows another principle.
+
+<div class="practice">
+
+<p><span class="practicelab" id="principle-pareto-frontier">When confronted with an
+apparent tradeoff, first look for ways to improve all principles at once.</span></p>
+
+Given any initial design that doesn't perfectly satisfy all principles, there are usually
+some other designs that improve the situation for some principles without sacrificing
+anything about the other principles. Work to find those designs.
+
+Another way to say this is to look for [Pareto
+improvements](https://en.wikipedia.org/wiki/Pareto_efficiency) before starting to trade
+off between principles.
+
+</div>
+
+Once one is choosing between different designs at the Pareto frontier, the choice of which
+privacy principles to prefer is complex and depends heavily on the details of each
+particular situation. <span class="note">Note that people's privacy can also be in tension
+with other important non-privacy concerns. As discussed in the [[[Ethical-Web]]], it's
+important to balance these concerns too, but this document doesn't discuss how to do
```

Comment:
Consider "... can also be in tension with other non-privacy concerns. As discussed in the [[[Ethical-Web]]], it's "... important to consider the context in which a particular technology is being applied, the expected audience(s) for the technology, who the technology benefits and who it may disadvantage, and any power dynamics involved..."

(words come from the Ethical Web Principles)

Reply:
Done.